zebra-chain/src/transaction/tests/vectors.rs

```diff
@@ -1,13 +1,32 @@
+use lazy_static::lazy_static;
+
+use zebra_test::zip0244;
+
 use super::super::*;
 
 use crate::{
     block::{Block, Height, MAX_BLOCK_BYTES},
     parameters::{Network, NetworkUpgrade},
     serialization::{SerializationError, ZcashDeserialize, ZcashDeserializeInto, ZcashSerialize},
+    transaction::txidhash::TxIdHasher,
 };
 
+use color_eyre::eyre::Result;
+
 use std::convert::TryInto;
```

Comment:
Nit: Maybe join some import lines?

```suggestion
use std::convert::TryInto;

use color_eyre::eyre::Result;
use lazy_static::lazy_static;

use zebra_test::zip0244;

use super::super::*;
use crate::{
    block::{Block, Height, MAX_BLOCK_BYTES},
    parameters::{Network, NetworkUpgrade},
    serialization::{SerializationError, ZcashDeserialize, ZcashDeserializeInto, ZcashSerialize},
    transaction::txidhash::TxIdHasher,
};
```

Reply:
Thanks! I need to pay more attention to the imports